credential/zkpass/v0.1.0/schema.json

```diff
@@ -0,0 +1,91 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://common.schemas.verida.io/credential/zkpass/v0.1.0/schema.json",
+    "title": "Schema for ZKPass",
+    "description": "A schema for zkpass credential",
+    "type": "object",
+    "appearance": {
+        "style": {
+            "color": "#4bbf95",
+            "icon": "./icon.svg"
+        }
+    },
+    "database": {
+        "name": "zkpass_credentials",
```

Comment:
Normally we specify a database, however this is a credential schema which doesn't get stored directly.

This credential data gets wrapped up in a separate Verida credential storage schema which specifies the database where it is stored.

So in this case you can remove the `database` section entirely.